mediapipe/graphs/instantmotiontracking/instantmotiontracking.pbtxt

```diff
@@ -1,13 +1,77 @@
-# MediaPipe graph that displays the camera view, boilerplate template for a graph
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-# Images coming into and out of the graph.
+# MediaPipe graph that performs region tracking and 3d object (AR sticker) rendering.
+
+# Images in/out of graph with sticker data and IMU information from device
 input_stream: "input_video"
-output_stream: "input_video"
+input_stream: "sticker_data_string"
+input_stream: "imu_data"
+output_stream: "output_video"
+
+# Converts sticker data into user data (rotations/scalings), render data, and
+# initial anchors.
+node {
+  calculator: "StickerManagerCalculator"
+  input_stream: "STRING:sticker_data_string"
+  output_stream: "ANCHORS:initial_anchor_data"
+  output_stream: "USER_ROTATIONS:user_rotation_data"
+  output_stream: "USER_SCALINGS:user_scaling_data"
+  output_stream: "RENDER_DATA:sticker_render_data"
+}
+
+# Uses box tracking in order to create 'anchors' for associated 3d stickers.
+node {
+  calculator: "RegionTrackingSubgraph"
+  input_stream: "VIDEO:input_video"
+  input_stream: "ANCHORS:initial_anchor_data"
+  output_stream: "ANCHORS:tracked_scaled_anchor_data"
+}
+
+# Finalizes all translation data and converts coordinates from normalized device
+# system to OpenGL coordinates.
+node {
+  calculator: "TranslationManagerCalculator"
+  input_stream: "ANCHORS:tracked_scaled_anchor_data"
+  input_stream: "USER_SCALINGS:user_scaling_data"
```

Comment:
I recommend separating user scaling here, so that this calculator only takes care of 3D translation induced by tracking. Feed user_scaling to MatricesManagerCalculator. Scaling can be represented by a right multiply with a diagonal matrix.